CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 0.2.8 - 2021-10-12
+### Changed
+- Refactored semantic providers [#59](https://github.com/EthTx/ethtx/pull/59)
+- `Signatures` collection in **pymongo** is now indexed [#59](https://github.com/EthTx/ethtx/pull/59)
+
+### Fixed
+- Fixed unknown args for empty guessed functions (empty arguments have been removed) [#59](https://github.com/EthTx/ethtx/pull/59)
+- Wrong `Signatures` collection indexing. [#60](https://github.com/EthTx/ethtx/pull/60)
```

Comment:
1. in mongo, not pymongo
2. Think one of `Wrong `Signatures` collection indexing.`  or ``Signatures` collection in **pymongo** is now indexed` should be removed